NPB3.3.1/NPB3.3-OMP/config/suite.def

```diff
@@ -0,0 +1,50 @@
+# config/suite.def
+# This file is used to build several benchmarks with a single command.
```

Comment:
Does it make sense to add this file to the npb-hooks repo?

Reply:
Since this file was already in the npb-hooks repo, so I thought of modifying it further in the source repo according to our needs. Originally, in npb tutorial (when I was using npb source from their official website), I had added instructions to explicitly add this file and transfer to the disk through packer. So, I can revert back to that if that makes more sense